content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,222 @@
+
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+
+      <div class="col-md-8">
+<!--
+        <h3>Objectives</h3>
+-->
+        <h3>目标</h3>
+        <ul>
+<!--
+          <li>Learn about Kubernetes Pods.</li>
+          <li>Learn about Kubernetes Nodes.</li>
+          <li>Troubleshoot deployed applications.</li>
+-->
+          <li>了解 Kubernetes Pod.</li>
+          <li>了解 Kubernetes Node.</li>
+          <li>对已部署的应用故障排除</li>
+        </ul>
+      </div>
+
+      <div class="col-md-8">
+<!--
+        <h2>Kubernetes Pods</h2>
+-->
+        <h2> Kubernetes Pods </h2>
+<!--
+        <p>When you created a Deployment in Module <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, Kubernetes created a <b>Pod</b> to host your application instance. A Pod is a Kubernetes abstraction that represents a group of one or more application containers (such as Docker or rkt), and some shared resources for those containers. Those resources include:</p>
+-->
+        <p>在模块 <a href="/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>创建 Deployment 时, Kubernetes 添加 a <b>Pod</b> 托管你的应用实例。Pod 是 Kubernetes 抽象出来的，表示一组一个或多个应用程序容器（如 Docker 或 rkt ），以及这些容器的一些共享资源。这些资源包括:</p>
+<!--
+        <ul>
+          <li>Shared storage, as Volumes</li>
+          <li>Networking, as a unique cluster IP address</li>
+          <li>Information about how to run each container, such as the container image version or specific ports to use</li>
+        </ul>
+-->
+        <ul>
+          <li>共享存储，当作卷</li>
+          <li>网络，作为唯一的集群 IP 地址</li>
+          <li>有关每个容器如何运行的信息，例如容器映像版本或要使用的特定端口。</li>
+        </ul>
+    <!--
+        <p>A Pod models an application-specific "logical host" and can contain different application containers which are relatively tightly coupled. For example, a Pod might include both the container with your Node.js app as well as a different container that feeds the data to be published by the Node.js webserver. The containers in a Pod share an IP Address and port space, are always co-located and co-scheduled, and run in a shared context on the same Node.</p>
+    -->
+        <p>Pod 为特定于应用程序的“逻辑主机”建模，并且可以包含相对紧耦合的不同应用容器。例如，Pod 可能既包含带有 Node.js 应用的容器，也包含另一个不同的容器，用于提供 Node.js 网络服务器要发布的数据。Pod 中的容器共享 IP 地址和端口，始终位于同一位置并且共同调度，并在同一节点上的共享上下文中运行。</p>
+<!--
+        <p>Pods are the atomic unit on the Kubernetes platform. When we create a Deployment on Kubernetes, that Deployment creates Pods with containers inside them (as opposed to creating containers directly). Each Pod is tied to the Node where it is scheduled, and remains there until termination (according to restart policy) or deletion. In case of a Node failure, identical Pods are scheduled on other available Nodes in the cluster.</p>
+-->
+        <p>Pod是 Kubernetes 平台上的最小单元。 当我们在 Kubernetes 上创建 Deployment 时，该 Deployment 会在其中创建包含容器的 Pod （而不是直接创建容器）。每个 Pod 都与调度它的节点绑定，并保持在那里直到终止（根据重启策略）或删除。 如果节点发生故障，则会在群集中的其他可用节点上调度相同的 Pod。</p>
```

Comment:
原子单元 可能更好点，更多的是说 不可再分

```suggestion
        <p>Pod是 Kubernetes 平台上的原子单元。 当我们在 Kubernetes 上创建 Deployment 时，该 Deployment 会在其中创建包含容器的 Pod （而不是直接创建容器）。每个 Pod 都与被调度部署的工作节点绑定，并保持在那里直到终止（根据重启策略）或删除。 如果工作节点发生故障，则会在群集中的其他可用工作节点上调度相同的 Pod。</p>
```